nmigen_boards/zturn_lite.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.xilinx_7series import *
+from .dev import *
+
+__all__ = ["ZTurnLitePlatform"]
+
+
+class ZTurnLitePlatform(Xilinx7SeriesPlatform):
+    device = "xc7z010"
+    package = "clg400"
+    speed = "1"
+
+    resources = []
```

Comment:
I'm not sure I want to merge this as-is -- no peripherals at all?

Reply:
the only external peripherals are connected to the cpu, so they have to be accessed through the `ps7` block